tests/test_cli.py

```diff
@@ -528,3 +528,15 @@ def test_info_cli(run_cli, tmpdir):
         "DHuS version: 2.4.1\n" in result.output
     )
     tmpdir.remove()
+
+
+@pytest.mark.vcr
+@pytest.mark.scihub
+def test_location_cli(run_cli, tmpdir):
+    result = run_cli("--location", "Metz", "-l", "5")
+    assert (
+        "Found 23106 products\nThe location we are querying is: 'Metz, Moselle, Grand Est, France métropolitaine, "
```

Comment:
You can be a bit less specific on the test success criterion. E.g. just verify that "Found" is in the output, not the full message. Seems like your test is currently failing on some details in the text.